x-pack/plugins/cases/server/features_v2.ts

```diff
@@ -0,0 +1,195 @@
+    order: FEATURE_ORDER,
+    management: {
+      insightsAndAlerting: [APP_ID],
+    },
+    cases: [APP_ID],
+    privileges: {
+      all: {
+        api: apiTags.all,
+        cases: {
+          create: [APP_ID],
+          read: [APP_ID],
+          update: [APP_ID],
+          push: [APP_ID],
+        },
+        management: {
+          insightsAndAlerting: [APP_ID],
+        },
+        savedObject: {
+          all: [...filesSavedObjectTypes],
+          read: [...filesSavedObjectTypes],
+        },
+        ui: capabilities.all,
+      },
+      read: {
+        api: apiTags.read,
+        cases: {
+          read: [APP_ID],
+        },
+        management: {
+          insightsAndAlerting: [APP_ID],
+        },
+        savedObject: {
+          all: [],
+          read: [...filesSavedObjectTypes],
+        },
+        ui: capabilities.read,
+      },
+    },
+    subFeatures: [
+      {
+        name: i18n.translate('xpack.cases.features.deleteSubFeatureName', {
+          defaultMessage: 'Delete',
+        }),
+        privilegeGroups: [
+          {
+            groupType: 'independent',
+            privileges: [
+              {
+                api: apiTags.delete,
+                id: 'cases_delete_v2',
+                name: i18n.translate('xpack.cases.features.deleteSubFeatureDetails', {
+                  defaultMessage: 'Delete cases and comments',
+                }),
+                includeIn: 'all',
+                savedObject: {
+                  all: [...filesSavedObjectTypes],
+                  read: [...filesSavedObjectTypes],
+                },
+                cases: {
+                  delete: [APP_ID],
+                },
+                ui: capabilities.delete,
+              },
+            ],
+          },
+        ],
+      },
+      {
+        name: i18n.translate('xpack.cases.features.casesSettingsSubFeatureName', {
+          defaultMessage: 'Case settings',
+        }),
+        privilegeGroups: [
+          {
+            groupType: 'independent',
+            privileges: [
+              {
+                id: 'cases_settings_v2',
+                name: i18n.translate('xpack.cases.features.casesSettingsSubFeatureDetails', {
+                  defaultMessage: 'Edit case settings',
+                }),
+                includeIn: 'all',
+                savedObject: {
+                  all: [...filesSavedObjectTypes],
+                  read: [...filesSavedObjectTypes],
+                },
+                cases: {
+                  settings: [APP_ID],
+                },
+                ui: capabilities.settings,
+              },
+            ],
+          },
+        ],
+      },
+      {
+        name: i18n.translate('xpack.cases.features.addCommentsSubFeatureName', {
+          defaultMessage: 'Add comments',
+        }),
+        privilegeGroups: [
+          {
+            groupType: 'independent',
+            privileges: [
+              {
+                api: apiTags.all,
```

Comment:
I think we should split up the tags in `x-pack/plugins/cases/common/utils/api_tags.ts` like:

1. `all` should not contain the `create` (create file tags).
2. Along with `all`, `read`, and `delete` we should return `createComment` and put the `create` (maybe rename it to createFileTags)

Then in this sub-feature privilege use `apiTags.createComment`.